src/main/java/org/myrobotlab/service/AudioFile.java

```diff
@@ -275,10 +268,12 @@ public AudioData waitFor(String filename, Object waitForMe) {
   }
 
   public void stop() {
+    AudioProcessor ap = processors.get(currentTrack);
     // dump the current song
-    processors.get(currentTrack).isPlaying = false; // FIXME -- necessary ????
+   
     // pause the next one if queued
-    processors.get(currentTrack).pause(false);
+    ap.pause(false); // FIXME me shouldn't it be true ?

```

Comment:
why do we pass a boolean into the pause method at all?   it should probably be refactored to simply be  ap.pause()  ...

Reply:
well you need a resume() then ..
6 to 1/2 dozen ...

Reply:
resume makes sense to me ... seems intuitive.